app/models/supervisor_volunteer.rb

```diff
@@ -7,6 +7,14 @@ class SupervisorVolunteer < ApplicationRecord
   validates :volunteer_id, uniqueness: {scope: :is_active}, if: :is_active?
   validate :ensure_supervisor_and_volunteer_belong_to_same_casa_org, if: -> { supervisor.present? && volunteer.present? }
 
+  scope :supervisor_volunteers_learning_hours, ->(supervisor_id) {
```

Comment:
remember to write a test making sure that supervisors for one casa_org cannot see any data relating to a different casa_org :)

Reply:
Have opted to use `LearningHour` instead so we no longer need this 😄